data/pdfarranger.ui

```diff
@@ -19,318 +18,323 @@ You should have received a copy of the GNU General Public License
 along with PDF Arranger.  If not, see <http://www.gnu.org/licenses/>.
 -->
 <interface>
-  <requires lib="gtk+" version="3.12"/>
-  <object class="GtkApplicationWindow" id="main_window">
+  <object class="HdyApplicationWindow" id="main_window">
     <property name="show_menubar">False</property>
-    <child type="titlebar">
-      <object class="GtkHeaderBar" id="header_bar">
+    <child>
+      <object class="GtkBox">
         <property name="visible">True</property>
-        <property name="spacing">4</property>
-        <property name="show_close_button">True</property>
+        <property name="orientation">vertical</property>
```

Comment:
This new GtkBox causes the indentation for the rest of the file to increase. Locally the diff is the same, so I don't think I can do much more.